### DISCUSSION TITLE
What is the difference between the Supply Chain Factors and the published USEEIO model results?

### DISCUSSION THREAD

**Discussion by MoLi7** [USEPA/supply-chain-factors#10] [Q&A] [answered]
It looks like they present similar results.

**Comment by MoLi7** [comment]
There are two major differences between the two datasets:

1. The Supply Chain Factors are in **purchaser** price, whereas the published USEEIO model results are in **producer** price.
2. The Supply Chain Factors do not convert major GHGs (i.e. CO2, CH4, N2O) into CO2e, whereas the published USEEIO model results report all GHGs in CO2e.

**Reply by WesIngwersen** [reply]
3. The Supply Chain factors v1 were adjusted to be in 2018 USD. USEEIO v2.0 model matrices are in 2012 USD.

**Reply by WesIngwersen** [reply]
4. The SEF v1 releases are built from a different USEEIO models that vary by sector level, GHG and IO data years, and commodity vs. industry type. See the [Appendix 2 - USEEIO Model Descriptions](https://cfpub.epa.gov/si/si_public_file_download.cfm?p_download_id=540830&Lab=CESER) for a full description of the different models used.

**Reply by Brian-Solomon** [reply]
Thank you Wes. Am I understanding correctly that the USEEIO model results (contained in USEEIO v2.0.1-411 and USEEIOv1.2; accessible via the link below) do NOT provide the MEFs (i.e., only SEFs)? And also, which GWP values are being used in the USEEIO model results to report GHGs in CO2e (e.g., IPCC Fifth Assessment Report)? Thank you. 
https://www.epa.gov/land-research/us-environmentally-extended-input-output-useeio-technical-content

**Reply by MoLi7** [reply]
Hi @Brian-Solomon, you are correct that USEEIO models only provide SEFs, as their results are in producer price. AR5 was the GWP values used in USEEIO v2 models.

**Reply by mksheridan** [reply]
Was AR4 or AR5 used in the USEEIO v1.1 model (https://catalog.data.gov/dataset/useeio-v1-1-matrices)?

**Reply by bl-young** [reply]
AR4 was used in v1.1 and is [still used in the latest release](https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.1.19-GHG.yml#L39).

**Reply by mksheridan** [reply]
Thank you! To confirm was AR4 or AR5 used in USEEIO v2.0.1-411 (https://catalog.data.gov/dataset/useeio-v2-0-1-411)?

**Reply by bl-young** [reply]
From EPA's TRACI, so AR4

**Comment by gsdipietro** [comment]


**Reply by bl-young** [reply]
Would you mind posting this as a [new discussion item](https://github.com/USEPA/supply-chain-factors/discussions) as it likely has value to others?

**Comment by WesIngwersen** [comment, marked as answer]
The major differences are:

1. The Supply Chain Factors are in  purchaser price, whereas raw values for GHGs/$ commodity in a USEEIO > v2 model $N$ matrix are in producer price
2. The Supply Chain Factors with Margins add on the GHGs of transportation, wholesaling and retailing. Results with margins can be calculates with a USEEIO model but are not given in published matrices.

One note - in Supply Chain Factor versions before v1.2, there were also no CO2e values given in the Supply Chain Factors, which was another difference with USEEIO models. This is no longer the case.
